services/supervisor/src/schema.rs

```diff
@@ -61,13 +61,28 @@ impl<'a, T: IndexAccess> Schema<'a, T> {
         )
     }
 
+    pub fn config_confirms(&self) -> ValidatorMultisig<T, Hash> {
+        ValidatorMultisig::new(
+            [self.instance_name, ".config_confirms"].concat(),
+            self.access.clone(),
+        )
+    }
+
+    pub fn config_propose_with_hash_entry(&self) -> Entry<T, ConfigProposeEntry> {
```

Comment:
Nit: Since this function returns the `Entry<T, ConfigProposeEntry>` I think that `config_propose_entry` will be a sufficient name.